solutions/hw1/task1/matrices.py

```diff
@@ -0,0 +1,41 @@
+from solutions.hw1.task1.vectors import Vector, get_vector_sum, scalar
+from typing import List
+import copy
+
+
+class Matrix:
+    content: List[Vector]
+
+    def __init__(self, first: Vector, *args: Vector):
+        if len(first) == 0:
+            raise TypeError("Matrix can't be created from empty vectors")
+        if not all(len(first) == len(vector) for vector in args):
+            raise TypeError("Rows must be the same length")
+        self.content = [first, *args]
+
+    def __eq__(self, other):
```

Comment:
You can also add typing (of arguments and output) in these functions (`eq`, `add`, `get_dimensions`, `mul`) too.